sleap_roots/bases.py

```diff
@@ -12,8 +12,15 @@ def get_bases(pts: np.ndarray) -> np.ndarray:
     Returns:
         Array of bases (instances, (x, y)).
     """
-    # (instances, 2)
-    base_pts = pts[:, 0]
+    # Exceptions
+    if len(pts) == 0 or np.isnan(pts[:, 0].all()):
+        # (instances, 2)
+        base_pts = np.empty((0, 2))
```

Comment:
Document that this is the shape that will be returned in this case by describing it in the Returns section of the docstring

Reply:
Done